events/static/js/app/ModalWindow.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import ChipInput from 'material-ui-chip-input'
+import Avatar from 'material-ui/Avatar'
+import Chip from 'material-ui/Chip';
+import blue300 from 'material-ui/styles/colors';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+
+export default class ModalWindow extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.handleAddChip = this.handleAddChip.bind(this);
+        this.handleDeleteChip = this.handleDeleteChip.bind(this);
+    }
+
+    handleAddChip(chip) {
+        if (this.props.dataSource.indexOf(chip) != -1) {
+           let chips = this.props.dataDestination;
+            chips.push(chip);
+            this.props.handleToSendList(chips);
+        }
+    };
+
+    handleDeleteChip(chip) {
+        let chips = this.props.dataDestination;
+            chips.splice(chip.indexOf(chip), 1);
+            this.props.handleToSendList(chips);
+    }
+
+    render() {
+
+        const dataSourceConfig = {
+            text: 'username',
+            value: 'username'
+        };
+
+        const standardActions = [
+            <FlatButton
+                label="Cancel"
+                primary={true}
+                onTouchTap={this.props.onCancelClose}
+            />,
+            <FlatButton
+                label="Submit"
+                primary={true}
+                onTouchTap={this.props.onSubmitClose}
+                disabled={(this.props.dataDestination.length == 0)}
+            />,
+        ];
+
+        const alertActions = (
+            <FlatButton
+                label="OK"
+                primary={true}
+                onTouchTap={this.props.onCancelClose}
+            />
+        );
+
+        const styles = {
+            dialogTitle: {
+                backgroundColor: 'rgb(0, 151, 167)',
```

Comment:
@yevgenbehen it has been changed in next pull request:[https://github.com/lhalam/event_manager/pull/19](url)